wolfssl/wolfcrypt/settings.h

```diff
@@ -3168,6 +3168,14 @@ extern void uITRON4_free(void *p) ;
     #undef NO_DH
 #endif
 
+/* CryptoCell defines */
+#ifdef WOLFSSL_CRYPTOCELL
```

Comment:
I think you may have lost the `WOLFSSL_ASYNC_CRYPT` one? This section can just be about which ECC signing hardware support zero digest.

Reply:
@dgarske I added it later on in the WOLFSSL_ASYNC_CRYPT section of settings.h, see line [3206](https://github.com/wolfSSL/wolfssl/pull/8118/files#diff-9efb2cca94d744438b9691461eb2bf010ecf7b2e9a6e24c8d6fd7872a663197aR3206).

That is fair, I was thinking I'd add the disable to the section of settings.h that corresponds to each higher level feature that wants to disable it, since that is what would drive the decision.

That way all the dependent options for ASYNC are set in the ASYNC section, the dependent options for cryptocell are set in the cryptocell section, etc. I noticed there wasn't a centralized cryptocell section for settings.h, so I created one.

Let me know if you would rather have it relocated to the ECC section?